src/reducers/filesReducer.js

```diff
@@ -2,7 +2,8 @@ import {
   UPDATE_A_FILE_SUCCESS,
   FETCH_FILES_SUCCESS,
   ADD_A_FILE_SUCCESS,
-  DELETE_A_FILE_SUCCESS
+  DELETE_A_FILE_SUCCESS,
```

Comment:
RETRIEVE_FILE_SUCCESS is not dispatched, how do you get files ?
add a test for RETRIEVE_DIR_SUCCESS with some files in action.response.files